docs/content/en/docs/architecture/monitoring/_index.md

```diff
@@ -6,24 +6,39 @@ description: >
+
+![Monitoring Architecture](monitoring-architecture.png)
+
+Let's walk through this architecture from left to right. We'll provide links to the Kubernetes documentation so you can dig into those concepts more if you'd like to.
 
-* Prometheus `ServiceMonitor` resource with Metric Relabeling directed at the
-  K8ssandra cluster's Service
-* Grafana `DataSource` resource configured to reference the deployed Prometheus
-  instance
-* Grafana `Dashboard` resources
+* The Cassandra nodes in a K8ssandra-managed cluster are organized in one or more data centers, each of which is represented as a Kubernetes [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/). (We'll focus here on details of the Cassandra node related to monitoring, you can see other details about Cassandra nodes such as how storage is managed on the [Cassandra architecture](/docs/architecture/cassandra) page.)
 
-![Grafana Cluster](grafana-cluster.png)
+* Each Cassandra node is deployed as its own [pod](https://kubernetes.io/docs/concepts/workloads/pods/). The pod runs the Cassandra daemon in a Java VM. Each Apache Cassandra pod is configured with the DataStax [Metrics Collector for Apache Cassandra](https://github.com/datastax/metric-collector-for-apache-cassandra), which is implemented as a Java agent running in that same VM. The Metrics Collector is configured to expose metrics on the standard Prometheus port (9103).
 
-## Metric Data Retention
+* One or more Prometheus instances are deployed in another StatefulSet, with the default configuration starting with a single instance. Using a StatefulSet allows each Prometheus node to connect to a Persistent Volume (PV) for longer term storage. The default K8ssandra chart configuration does not use PVs. By default, metric data collected in the cluster is retained within Prometheus for 24 hours.
 
-Metric data collected in the cluster is retained within Prometheus for 24 hours.
+* An instance of the Prometheus Operator is deployed using a Replica Set. The `kube-prometheus-stack` also defines several useful Kubernetes [custom resources (CRDs)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) that the Prometheus Operator uses to manage Prometheus. One of these is the `ServiceMonitor`. K8ssandra uses `ServiceMonitor` resources, specifying labels selectors to indicate the Cassandra nodes to connect to in each datacenter, and how to relabel each metric as it is stored in Prometheus. `ServiceMonitors` can also be configured to pull metrics from `cass-operator` and Stargate.
+
+* The `AlertManager` is an additional resource provided by `kube-prometheus-stack` that can be configured to specify thresholds for specific metrics that will trigger alerts. Use of this feature is a K8ssandra [roadmap](/docs/roadmap) item. 
```

Comment:
Users may enable, and configure, AlertManager through the `values.yaml` file. Maybe we should direct them to check the options out at https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L114-L595?